cmd/snap/cmd_interfaces.go

```diff
@@ -86,6 +87,10 @@ func (x *cmdInterfaces) Execute(args []string) error {
 	if len(ifaces.Plugs) == 0 && len(ifaces.Slots) == 0 {
 		return fmt.Errorf(i18n.G("no interfaces found"))
 	}
+
+	defer fmt.Fprintf(Stderr, "\n%s\n",
+		i18n.G("The 'snap interfaces' command is deprecated, try the new 'snap connections'."))
```

Comment:
```suggestion
		i18n.G("The 'snap interfaces' command is deprecated, try the new 'snap connections' command."))
```

Reply:
Or maybe add a "please":  `	i18n.G("The 'snap interfaces' command is deprecated, please try the new 'snap connections' command."))` maybe a question for @degville

Reply:
I'm always over minimal in these situations, as I think the user simply needs to quickly scan the facts - I'd even remove *new* (because when is new?), and *try* because we need to be confident. We can always explain more on the forum.
I'd put:
`The 'snap interfaces' command is deprecated. Use 'snap connections' instead.`

Reply:
Sounds good to me. Thanks for the suggestion @degville !